en/px-demographics.json

```diff
@@ -713,9 +713,6 @@
                 "question": "Which of these is the carrier for the mobile phone you use most often?",
                 "shortName": "Carrier for the mobile phone you use most often"
             },
-            "noPhone": {
```

Comment:
is this currently used?

Reply:
It will be deprecated with the related migrations PR. but it's true perhaps we should keep this for the time being.

Reply:
If it's used on PRO we have to keep it in the translations until we are no longer using it

Reply:
I understand, makes sense